docs/volunteering/communication-platforms.md

```diff
@@ -0,0 +1,33 @@
+# Communication Platforms
+
+## Slack
+
+Slack is a realtime messaging platform which allows for synchronous discussion
+in channels and threads.
+
+We use it for discussion, though (due to the nature of the platform) it has
+limitations in this regard.
+
+We occasionally announce things via Slack, however anything important will also
+be sent via email.
+
```

Comment:
I'd like to add the link to the signup form here, not sure if that's inviting spam though?

Reply:
5a2eabfd923bc6ec64cb47fb96bdddee00995599